src/scripts.js

```diff
@@ -78,33 +85,89 @@ const parseData = (data) => {
   //below functions will move to domManip on click events
   session.getTotalDollarSpentThisYear()
   session.getTripCost('Cartagena, Colombia', 7, 2)
+  addDestinationSearch()
   // session.getPastTrips(todayDate)
-  changeFormView()
+  // changeFormView(session, todayDate)
+  // displayPastTrips()
   // session.getUpcomingTrips(todayDate)
   // toggleView()
   // renderDom(data)
 }
 
+
+
 const changeFormView = () => {
   switch (tripView.value) {
   case 'past':
-    session.getPastTrips(todayDate)
+
+    let past = session.getPastTrips(todayDate)
+    displayTrips(past, tripView.value)
     break;
   case 'upcoming':
-    session.getUpcomingTrips(todayDate)
-
+    let upcoming = session.getUpcomingTrips(todayDate)
+    displayTrips(upcoming, tripView.value)
     break;
   case 'pending':
-    session.getPendingTrips()
+    let pending = session.getPendingTrips()
+    displayTrips(pending, tripView.value)
     break;
-  //   case 'expenses.':
-  //
-  //     break;
+    // case 'expenses.':
+    //
+    //   break;
   default:
     console.log('something went wrong');
     break;
   }
 };
+
+const displayTrips = (trips, value) => {
+  if (trips.length > 0) {
+    bottomHeading.innerHTML =
+    '<h2>my ' + value + ' trips</h2>'
+
+    destinationsSection.innerHTML = ''
+    trips.forEach(trip => {
+      session.destinationData.forEach(dest => {
+        if (trip.destinationID === dest.id) {
+          destinationsSection.innerHTML += `
+          <section class='trips-display'>
+          <h4>${dest.destination}</h4>
+          <p>${trip.date}</p>
+          </section>
+          `
+        }
+      })
+    })
+  } else {
+    bottomHeading.innerHTML = '<p>You have no ' + value + ' trips at this time.</p>'
+    destinationsSection.innerHTML = ''
+  }
+}
+
+
+
+const displayUpcomingTrips = () => {
+  bottomHeading.innerHTML = `
+  <h2>my upcoming trips</h2>
+  `
+  destinationsSection.innerHTML = `
+  `
+}
+
+const displayPendingTrips = () => {
+  bottomHeading.innerHTML = `
+  <h2>my pending trips</h2>
+  `
+  destinationsSection.innerHTML = `
```

Comment:
same note here as 153 about the back tics.

Reply:
thank you - making the changes on my local repo!